src/scripts/towns/Town.ts

```diff
@@ -192,10 +197,19 @@ TownList['Mt Silver'] = new DungeonTown('Mt Silver', [new RouteKillRequirement(1
 const LittleRootTownShop = new Shop([
     ItemList['Pokeball'],
 ]);
+const RustboroCityShop = new Shop([
+    ItemList['Mystery_egg'],
+]);
+const DewfordTownShop = new Shop([
+    ItemList['Fighting_egg'],
+]);
 const SlateportCityShop = new Shop([
     ItemList['Greatball'],
     ItemList['Water_egg'],
 ]);
+const MauvilleCityShop = new Shop([
+    ItemList['Electric_egg'],
+]);
```

Comment:
I think you may have forgotten to add this to the town 😅

Reply:
Gotta love that Travis is still fine with it. Other way around and Travis throws a fit. Can't have a shop added to a town only, but not added below //region shops.